src/grammar.pest

```diff
@@ -167,7 +167,7 @@
 
 // atomic value types
 
-    number = @{ "-"? ~ (number_leading | number_trailing) }
+    number = @{ number_leading | number_trailing }
```

Comment:
In the process, I found this confusing bit. Since the prefix/postfix rule has the highest priority, this negative sign will never be considered part of a "number" - it will always be a subtraction prefix unary op.

https://github.com/dgkf/R/blob/e440cf7f47f9778fb22bbf65331f6dbb5634fe8f/src/parser.rs#L101-L104

No need to keep it around and add further confusion.